soccer/Referee.cpp

```diff
@@ -212,13 +211,15 @@ void NewRefereeModule::run() {
     }
 }
 
-void NewRefereeModule::overrideTeam(bool isBlue) {
+void Referee::overrideTeam(bool isBlue) {
     if (!_isRefControlled) {
         blueTeam(isBlue);
     }
 }
```

Comment:
Shouldn't it just pull this information from the game state, and put it into the game state if we're getting it from the ref box?

That said, I think we need to discuss more how the referee module interacts with the GUI. There's some annoying override logic like this spread out in a lot of different places, and I'd like to figure out how best to consolidate/get rid of it (@chachmu).